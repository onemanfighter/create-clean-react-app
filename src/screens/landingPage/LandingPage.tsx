import { TitleBoxContainer } from '@components';
import { ThemeIcon } from '@components';

const LandingPage = () => {
  return (
    <TitleBoxContainer
      title={'Template-boilerplate'}
      icon="app"
      display="flex"
      flexDir="column"
      rowGap={10}
      height={'100vh'}
      paddingX={10}
    >
      <ThemeIcon />
    </TitleBoxContainer>
  );
};

export default LandingPage;
