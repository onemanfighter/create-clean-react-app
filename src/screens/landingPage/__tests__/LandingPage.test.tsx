import LandingPage from '../LandingPage';
import { renderWithRouter } from '@testUtils';

describe('LandingPage', () => {
  it('should render correctly', () => {
    const { container } = renderWithRouter(<LandingPage />);

    expect(container).toMatchSnapshot();
  });
});
