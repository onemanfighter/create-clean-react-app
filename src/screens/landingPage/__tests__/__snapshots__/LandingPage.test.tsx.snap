// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage should render correctly 1`] = `
<div>
  <div
    class="css-1wgbimk"
  >
    <div>
      <title>
        Template-boilerplate
      </title>
    </div>
    <button
      aria-label="instagram-icon"
      class="chakra-button css-1ehqmgk"
      type="button"
    >
      <svg
        aria-hidden="true"
        fill="currentColor"
        focusable="false"
        height="20"
        viewBox="0 0 24 24"
        width="20"
      >
        <path
          d="M9.57 3.38a8 8 0 0010.4 10.4 1 1 0 011.31 1.3 10 10 0 11-13-13 1 1 0 011.3 1.3zM7.1 5.04a8 8 0 1011.2 11.23A10 10 0 017.08 5.04z"
        />
      </svg>
    </button>
  </div>
</div>
`;
